caddyhttp/httpserver/plugin.go

```diff
@@ -490,6 +490,8 @@ var directives = []string{
 	"hugo",      // github.com/hacdias/caddy-hugo
 	"mailout",   // github.com/SchumacherFM/mailout
 	"awslambda", // github.com/coopernurse/caddy-awslambda
+
```

Comment:
Okay. I have left it at the bottom of the chain for now. I tested it along with tls,log and prometheus plugins on my dev machine and it all works fine